tests/commons/glob/test_glob.py

```
@@ -0,0 +1,159 @@
+def test_visitGlobPaths() -> None:
+    assert visitGlobPaths
+    assert GlobMatchVisitor
+
+    class TestVisitor(GlobMatchVisitor):
+        def __init__(self) -> None:
+            self.visited = []
+
+        def visitExcluded(self, path: Path) -> None:
+            raise RuntimeError(
+                f"in this test we should not exclude path {path}"
+            )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)